src/gui/attributetable/qgsdualview.cpp

```diff
@@ -85,6 +85,8 @@ void QgsDualView::init( QgsVectorLayer* layer, QgsMapCanvas* mapCanvas, const Qg
   else
     columnBoxInit();
 
+  mTableView->resizeColumnsToContents();
```

Comment:
No the fix didn't work.  I suspect it's going to require something cleaver
because doing this on any normal size table kills QGIS.

On Mon, Aug 31, 2015 at 9:06 PM, Alessandro Pasotti <
notifications@github.com> wrote:

> In src/gui/attributetable/qgsdualview.cpp
> <https://github.com/qgis/QGIS/pull/2259#discussion_r38302206>:
>
> > @@ -85,6 +85,8 @@ void QgsDualView::init( QgsVectorLayer* layer, QgsMapCanvas* mapCanvas, const Qg
> >    else
> >      columnBoxInit();
> >
> > +  mTableView->resizeColumnsToContents();
>
> @NathanW2 <https://github.com/NathanW2>, did you solve this issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qgis/QGIS/pull/2259/files#r38302206>.
>